Resfeber/View Controllers/ExploreViewController.swift

```diff
@@ -0,0 +1,170 @@
+        button.setImage(image, for: .normal)
+        
+        return button
+    }()
+    
+    fileprivate let searchBar: UISearchBar = {
+        let sb = UISearchBar(frame: .zero)
+        sb.placeholder = "Search"
+        sb.searchBarStyle = .minimal
+        return sb
+    }()
+        
+    // MARK: - Lifecycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureViews()
+    }
+    
+    // MARK: - Selectors
+    
+    @objc fileprivate func profileImageTapped() {
+        print("DEBUG: profileImageTapped..")
+    }
+    
+    // MARK: - Helpers
+    
+    fileprivate func configureViews() {
+        view.backgroundColor = UIColor.Resfeber.background
+        navigationController?.navigationBar.isHidden = true
+        
+        // Configure Title Label
+        view.addSubview(titleLabel)
+        titleLabel.anchor(top: view.safeAreaLayoutGuide.topAnchor, left: view.leftAnchor,
+                          right: view.rightAnchor, paddingTop: 4, paddingLeft: 20, paddingRight: 20)
+        
+        // Configure Profile Button
+        view.addSubview(profileButton)
+        profileButton.anchor(right: view.rightAnchor, paddingRight: 20)
+        profileButton.centerY(inView: titleLabel)
+        
+        // Configure Search Bar
+        searchBar.delegate = self
+        view.addSubview(searchBar)
+        searchBar.anchor(top: titleLabel.bottomAnchor, left: view.leftAnchor, right: view.rightAnchor,
+                         paddingTop: 12, paddingLeft: 12, paddingRight: 12)
+        
+        // Configure Collection View
+        let layout = UICollectionViewFlowLayout()
+        collectionView = UICollectionView(frame: view.frame, collectionViewLayout: layout)
+        collectionView.backgroundColor = UIColor.Resfeber.background
+        collectionView.showsVerticalScrollIndicator = false
+        collectionView.register(DestinationCell.self, forCellWithReuseIdentifier: DestinationCell.reuseIdentifier)
+        collectionView.dataSource = self
+        collectionView.delegate = self
+        view.addSubview(collectionView)
+        collectionView.anchor(top: searchBar.bottomAnchor, left: view.leftAnchor, bottom: view.bottomAnchor,
+                              right: view.rightAnchor, paddingTop: 12, paddingLeft: 20, paddingRight: 20)
+        
+        // Load Data
+        DestinationController.readDestinations()
+        collectionView.reloadData()
+    }
+    
+    fileprivate func performQuery(with searchText: String?) {
+        let queryText = searchText ?? ""
+        print("DEBUG: Perform query with text: \(queryText)..")
+    }
+}
+
+// MARK: - Collection View Layout
+
+extension ExploreViewController: UICollectionViewDelegateFlowLayout {
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
+        let numberOfColumns: CGFloat = 2
```

Comment:
Do you always want this to be two? Or with larger size classes, do you want to increase it?

Reply:
I do always want it to be 2. For right now we just are developing the app for iPhone only. Landscape orientation will also be disabled.